Headlines/src/DataSources/ProfileDataSource.swift

```diff
@@ -90,49 +92,50 @@ class ProfileDataSource: NSObject, UICollectionViewDataSource {
             cell.label.text = interest.name
             cell.label.textColor = UIColor.black
             cell.backgroundColor = UIColor(white: 240/255.0, alpha: 1)
-            
+
             cell.normalTextColor = UIColor.black
             cell.normalBackgroundColor = UIColor(white: 240/255.0, alpha: 1)
             cell.selectedTextColor = UIColor.black
             cell.selectedBackgroundColor = UIColor(white: 240/255.0, alpha: 1)
-            
+
             return cell
-            
+
         case 1:
             guard let cell = collectionView.dequeueReusableCell(withReuseIdentifier: "reaction_cell", for: indexPath) as? ProfileReactionCollectionViewCell else {
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/Canillitapp/headlines-iOS/blob/f5a5ab936ee0d164c160af7374b637afef6eb821/Headlines/src/DataSources/ProfileDataSource.swift#L104"></span>Line should be 120 characters or less: currently 162 characters<br />
<code>line_length</code> <code>ProfileDataSource.swift:104</code></td>
    </tr>
  </tbody>
</table>